### PR TITLE
remove extra replaces

### PR DESCRIPTION
**Pull request by marbar3778**
Closes: #XXX

## Description

no need for these anymore

______

For contributor use:

- [ ] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/gaia/blob/main/CONTRIBUTING.md#pr-targeting))
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`)
- [ ] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [ ] Re-reviewed `Files changed` in the Github PR explorer

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -122,7 +123,5 @@ replace (
 	github.com/cosmos/cosmos-sdk => github.com/osmosis-labs/cosmos-sdk v0.43.0-rc3.0.20211209072213-711e78b4f6b4
 	github.com/cosmos/ibc-go/v2 => github.com/osmosis-labs/ibc-go/v2 v2.0.2-osmo
 	github.com/gogo/protobuf => github.com/regen-network/protobuf v1.3.3-alpha.regen.1
-	github.com/tendermint/tendermint => github.com/tendermint/tendermint v0.34.14
```
Comment (ValarDragon) [inline go.mod:125]: Thanks! This replace was based on my own misunderstanding of how go.mod / go.sum works =)

Comment (faddat) [reply go.mod:125]: lot of that going around :)

**Comment by ValarDragon** [comment]
Not sure how useful dependabot is going to be, e.g. I'll often want to hold back on patch versions, e.g. IAVL.

But worth a try, and removing it if its problematic!

**Comment by marbar3778** [comment]
> Not sure how useful dependabot is going to be, e.g. I'll often want to hold back on patch versions, e.g. IAVL.
> 
> But worth a try, and removing it if its problematic!

you can mark dependencies as don't update. This way you can update on your own

**Comment by ValarDragon** [comment]
ah cool!

**Pull request merged**